TransIT.DAL/Models/Entities/Part.cs

```diff
@@ -19,5 +20,8 @@ public class Part : IAuditableEntity, IBaseEntity
         public Manufacturer Manufacturer{ get; set; }
         public virtual User Create { get; set; }
         public virtual User Mod { get; set; }
+
+        public ICollection<IssueLogPart> IssueLogPart { get; set; }
+        public ICollection<SupplierPart> SupplierPart { get; set; }
```

Comment:
this property names for collections should not be IssueLogPart, should be IssueLogPart**s** and SupplierPart**s**